### PR TITLE
add auto-gen hd stuff

### PR DESCRIPTION
**Pull request by earthernsence**
```js
// semicolonsbecauseiforgottheminotherfilessohopefullythismakesupforit.js
for (let i = 0; i < 1e10; i++) {
;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
}
```

### CODE REVIEW

File: js/constants/automation.js [hunk 1]
```diff
@@ -56,44 +56,81 @@ const AUTOMATORS = {
 		return tmp.inf.upgs.has("4;2")||tmp.elm.bos.hasHiggs("0;0;0");
 	},
 	robots: function() {
-		return tmp.elm.bos.hasHiggs("1;0;0")
+		return tmp.elm.bos.hasHiggs("1;0;0");
```
Comment (earthernsence) [inline js/constants/automation.js:59]: I Think This Is A Bad Change

**Review by earthernsence — COMMENTED** [review]
Looks Good!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by earthernsence** [comment]
For this particular pull request, you'll probably have to force push. The amount of conflicts will be impossible to fix, and there won't be any issues with it.

**Comment by Jacorb90** [comment]
Well there are a bunch of conflicts on this, and pg132 actually already did the auto-gen stuff I think, so I'm gonna close this, sorry

**Comment by earthernsence** [comment]
I'll probably still go through and add a ton of semicolons on a separate branch, the code needs some work

**Pull request closed without merging**